scripts/helper_functions.sh

```diff
@@ -76,16 +76,31 @@ isExecutable() {
     return "$return_val"
 }
 
+# Convert player list from JSON format
+convert_JSON_to_CSV_players(){
+    echo 'name,playeruid,steamid'
+    echo -n "${1}" | \
+        jq -r '.players[] | [ .name, .playerId, .userId ] | @csv' | \
+        sed -re 's/"None"/"00000000000000000000000000000000"/' \
+        -re 's/"steam_/"/' \
+        -re 's/"//g'
+}
+
```

Comment:
### Memos
Input JSON example:
```json
{
        "players": [
                {
                        "name": "まっする",
                        "playerId": "3657AB7D000000000000000000000000",
                        "userId": "steam_76561198847285114",
                        "ip": "172.21.48.1",
                        "ping": 22.392856597900391,
                        "location_x": -359965,
                        "location_y": 268835,
                        "level": 1
                },
                {
                        "name": "muscle",
                        "playerId": "None",
                        "userId": "steam_76561198847285115",
                        "ip": "172.21.48.1",
                        "ping": 22.392856597900391,
                        "location_x": -359965,
                        "location_y": 268835,
                        "level": 1
                }
        ]
}
```
The "name" and "playerId" may be different when creating a character or logging in.

Output CSV example:
```csv
name,playeruid,steamid
まっする,3657AB7D000000000000000000000000,76561198847285114
muscle,00000000000000000000000000000000,76561198847285115
```